src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -117,10 +117,10 @@ public void InitializeService(ServiceHost serviceHost)
                 // if a connection string wasn't provided as a parameter then
                 // use the owner uri property to lookup its associated ConnectionInfo
                 // and then build a connection string out of that
-                ConnectionInfo connInfo;                                
-                ScriptingService.ConnectionServiceInstance.TryFindConnection(parameters.OwnerUri, out connInfo);
+                ConnectionInfo connInfo;
                 if (parameters.ConnectionString == null)
```

Comment:
maybe add `|| parameters.ScriptOptions.ScriptCreateDrop == "ScriptSelect` to this condition to avoid needing to duplicate the whole block again on line 138.